k2/csrc/fsa_algo.h

```diff
@@ -127,7 +128,7 @@ void Connect(const Fsa &a, Fsa *b, std::vector<int32_t> *arc_map = nullptr);
 void RmEpsilonsPruned(const WfsaWithFbWeights &a,
                       float beam,
                       Fsa *b,
-                      std::vector<std::vector> *arc_map);
+                      std::vector<std::vector<int32_t>> *arc_map);
```

Comment:
add `/*=nullptr*/` ?

Reply:
I am going to resolve all the conflicts this night out of working hours.

Reply:
This is the function signature and it does not have a default value.

`/*=nullptr*/ ` is useful only in the implementation file when  its default value is a nullptr.